eng/config/PublishData.json

```diff
@@ -213,26 +213,13 @@
       "insertionCreateDraftPR": true,
       "insertionTitlePrefix": "[d17.1p3]"
     },
-    "main-vs-deps": {
-      "nugetKind": [
-        "Shipping",
-        "NonShipping"
-      ],
-      "version": "4.1.*",
-      "packageFeeds": "default",
-      "channels": [],
-      "vsBranch": "main",
-      "vsMajorVersion": 17,
-      "insertionCreateDraftPR": true,
-      "insertionTitlePrefix": "[d17.2p1]"
-    },
     "main": {
       "nugetKind": [
         "Shipping",
         "NonShipping"
       ],
       "version": "4.1.*",
-      "packageFeeds": "arcade",
+      "packageFeeds": "default",
       "channels": [],
       "vsBranch": "main",
       "vsMajorVersion": 17,
```

Comment:
@Cosifne can't comment on the exact line, but on 239, we may want to change the prefix